rule.proto

```diff
@@ -79,4 +79,8 @@ message Schedule {
   oneof params {
     RepeatingParams repeating_params = 2;
   }
+
+  // required
+  // The timezone name of the scheduled job from the tz database.
```

Comment:
what is `tz` stands for?

Reply:
The tz stands for [time zone](https://en.wikipedia.org/wiki/Tz_database).  I'll add a link to the documentation to make this clearer.